### PR TITLE
Perenio PEHPL0X and PEHWE20 added

### PR DESCRIPTION
**Pull request by annar-dev**
The Perenio Power Link PEHPL0X and Perenio Mini-relay PEHWE20 added

### CODE REVIEW

File: devices/perenio.js [hunk 1]
```diff
@@ -1,7 +1,197 @@
...
+        },
+    },
+};
+
+const tzPerenio = {
+    switch_type: {
+        key: ['switch_type'],
+        convertSet: async (entity, key, value, meta) => {
+            const switchTypeLookup = {
+                'momentary-state': 0x0001,
```
Comment (Koenkk) [inline devices/perenio.js:118]: Please make the snake case (`-` -> `_`)

File: devices/perenio.js [hunk 2]
```diff
@@ -1,7 +1,197 @@
...
+
+const switchTypeValues = [
+    'maintained-state',
+    'maintained-toggle',
+    'momentary-state',
+    'momentary-press',
+    'momentary-release'
+];
+
+const alarmTypeValues = [
```
Comment (Koenkk) [inline devices/perenio.js:18]: These don't seem to be used ?

**Comment by Koenkk** [comment]
Thanks, could you also make a pr to add the device images to https://github.com/Koenkk/zigbee2mqtt.io/tree/master/public/images/devices ?

**Comment by annar-dev** [comment]
Thank you, the PR with device images was submitted: https://github.com/Koenkk/zigbee2mqtt.io/pull/1410

**Pull request merged**